### PR TITLE
Initial improving of code quality.

### PR DESCRIPTION
**Pull request by kegor**
Related PR (to get A-grade on Codacy) https://github.com/groupdocs-annotation/GroupDocs.Annotation-for-.NET-WebForms/pull/45.

### CODE REVIEW

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 43
- Added 1
           

Clones added
============
- src/Products/Signature/Signer/StampSigner.cs  3
- src/Products/Common/Util/Comparator/FileTypeComparator.cs  1
- src/Products/Common/Util/Comparator/FileDateComparator.cs  1
- src/Products/Common/Util/Comparator/FileNameComparator.cs  1
- src/Products/Signature/Signer/BarCodeSigner.cs  1
         

Clones removed
==============
+ src/Products/Signature/Controllers/SignatureApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-signature/GroupDocs.Signature-for-.NET-WebForms/pullRequest?prid=6336224&bid=20584276)

File: src/Products/Signature/Loader/SignatureLoader.cs [hunk 1]
```diff
@@ -264,9 +259,9 @@ public List<SignatureFileDescriptionEntity> LoadTextSignatures(string xmlFolder)
 
                 return fileList;
             }
-            catch (Exception ex)
+            catch (Exception)
```
Comment (egorovpavel) [inline src/Products/Signature/Loader/SignatureLoader.cs:262]: Codacy found an issue: [Add logic to this catch clause or eliminate it and rethrow the exception automatically.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Signature-for-.NET-WebForms/pullRequest?prid=6336225)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 43
- Added 1
           

Clones added
============
- src/Products/Common/Util/Comparator/FileTypeComparator.cs  1
- src/Products/Signature/Signer/StampSigner.cs  3
- src/Products/Common/Util/Comparator/FileNameComparator.cs  1
- src/Products/Common/Util/Comparator/FileDateComparator.cs  1
- src/Products/Signature/Signer/BarCodeSigner.cs  1
         

Clones removed
==============
+ src/Products/Signature/Controllers/SignatureApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Signature-for-.NET-WebForms/pullRequest?prid=6336225&bid=20584277)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 43
           

Clones added
============
- src/Products/Common/Util/Comparator/FileTypeComparator.cs  1
- src/Products/Signature/Signer/BarCodeSigner.cs  1
- src/Products/Common/Util/Comparator/FileDateComparator.cs  1
- src/Products/Signature/Signer/StampSigner.cs  3
- src/Products/Common/Util/Comparator/FileNameComparator.cs  1
         

Clones removed
==============
+ src/Products/Signature/Controllers/SignatureApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Signature-for-.NET-WebForms/pullRequest?prid=6336225&bid=20584277)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 43
           

Clones added
============
- src/Products/Common/Util/Comparator/FileTypeComparator.cs  1
- src/Products/Signature/Signer/BarCodeSigner.cs  1
- src/Products/Common/Util/Comparator/FileNameComparator.cs  1
- src/Products/Signature/Signer/StampSigner.cs  3
- src/Products/Common/Util/Comparator/FileDateComparator.cs  1
         

Clones removed
==============
+ src/Products/Signature/Controllers/SignatureApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-signature/GroupDocs.Signature-for-.NET-WebForms/pullRequest?prid=6336224&bid=20584276)

**Pull request merged**